packages/aws-rfdk/lib/deadline/lib/rq-connection.ts

```diff
@@ -111,6 +111,7 @@ export abstract class RenderQueueConnection {
         'if [ -f "/etc/profile.d/deadlineclient.sh" ]; then',
         '  source "/etc/profile.d/deadlineclient.sh"',
         'fi',
+        '"\${DEADLINE_PATH}/deadlinecommand" -SetIniFileSetting LaunchSlaveAtStartup True',
```

Comment:
We do not want to set this here. This class is a generic class used to set up any deadline isntance being able to connect to a render queue not specifically workers as such your changes make it so any Workstation, Pulse or other component automatically starts up workers.

This setting is also already being set in the ConfigureWorker.sh which is where I would expect all of this code to exist.

Reply:
After offline discussion we have found a way not to start worker before configuration Pools and Groups.